hack/workspaces_install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-export QUAY_NAMESPACE=${QUAY_NAMESPACE:-workspaces}
+export QUAY_NAMESPACE=${QUAY_NAMESPACE:-filario}
```

Comment:
Maybe fix this while we're at it?
```suggestion
export QUAY_NAMESPACE=${QUAY_NAMESPACE:-konflux-workspaces}
```

Reply:
done